js/store.js

```diff
@@ -360,6 +363,29 @@ const store = {
     // empty out request processing queue
     capture.queue = [];
     return await this.db.websites.clear();
+  },
+
+  async getOldestDate() {
+    const oldestSite = await this.db.websites.orderBy('requestTime').first();
+    if (!oldestSite) {
+      return '';
+    }
+    return oldestSite.requestTime;
+  },
+
+  async getNumSitesVisited() {
```

Comment:
Maybe this should be `getNumFirstParties` and leave "sites visited" as presentation only language rather than in the code.